app/models/casa_case.rb

```diff
@@ -97,6 +98,30 @@ def has_transitioned?
     transition_aged_youth
   end
 
+  def contains_emancipation_option?(option_id)
+    begin
+      emancipation_options.find(option_id)
+    rescue ActiveRecord::RecordNotFound
+      return false
+    end
+
+    true
+  end
```

Comment:
A simpler way to write this would be:

```
def contains_emancipation_option?(option_id)
  emancipation_options.find_by(id: option_id).present?
end
```